src/media/js/site/containers/login.js

```diff
@@ -4,9 +4,10 @@ import {bindActionCreators} from 'redux';
 
 import {fxaLoginBegin, login} from '../actions/login';
 import {LoginButton} from '../components/login';
+import {Page} from '../components/page';
 
 
-export class Login extends React.Component {
+export class LoginForm extends React.Component {
```

Comment:
I did that because "login" is an action that's taken, and a container should be a noun. I can change it back.